src/bin/travis/resources/local_conformance_expected_failures.txt

```diff
@@ -61,8 +61,8 @@
 61
 62
 63
-#64
```

Comment:
This is because it uses `MultipleInputFeatureRequirement` and was kinda working by accident before. It'll work for good with https://github.com/broadinstitute/cromwell/pull/3252

Reply:
>It'll work for good

Bold statement 😄